test/activities/CategoriesEntity.js

```diff
@@ -57,19 +57,21 @@ describe('CategoriesEntity', () => {
 			var categoriesEntity = new CategoriesEntity(nonEditableEntity);
```

Comment:
Hmm, I think for this test "when not present" we should have an entity that actually has no category selected, and expect `selected` to be undefined then.